docs/metrics/metrics-operators/eval.md

```diff
@@ -32,18 +31,18 @@ metrics query | eval expr([REDUCER BOOLEAN EXPRESSION | _value] [_granularity])
 
 **Example 1**
 
-This query returns the value of the cpu_idle metric, multiplied by 100.
+This query returns the value of the `CPU_Idle` metric, multiplied by 100.
 
 ```
-_sourceCategory=ApacheHttpServer metrics=cpu_idle | eval _value * 100
+_sourceCategory=ApacheHttpServer metric=CPU_Idle | eval _value * 100
```

Comment:
I'm wondering if  adding `_sourceCategory=ApacheHttpServer` selector actually adds value to the example - looks like this is a one-off in the examples provided